### ISSUE TITLE
This is not registration...

### ISSUE THREAD

**Issue by ivjia** [limingwu8/Image-Registration#2]
Hi, I have to say this is not image registration but a image synthesis/generation model. Registration means you predict the spacial transformation of the image and transform the image using interpolation, Not that there's no mapping between image intensity but just spacial transformations.

**Comment by limingwu8** [comment]
You are right. This is just a toy project for testing the image mapping ability of U-Net.
